config/internal/minio/route.yaml.tmpl

```diff
@@ -1,15 +1,18 @@
 kind: Route
 apiVersion: route.openshift.io/v1
 metadata:
-  name: minio-{{.Name}}
-  namespace: {{.Namespace}}
-  labels:
-    app: minio-{{.Name}}
-    component: data-science-pipelines
+ name: minio-{{.Name}}
+ namespace: {{.Namespace}}
+ labels:
+   app: minio-{{.Name}}
+   component: data-science-pipelines
 spec:
-  to:
-    kind: Service
-    name: minio-{{.Name}}
-    weight: 100
-  port:
-    targetPort: 9000
+ to:
+   kind: Service
+   name: minio-{{.Name}}
+   weight: 100
+ port:
+   targetPort: 9000
+ tls:
+   termination: Edge
+   insecureEdgeTerminationPolicy: Redirect
```

Comment:
the first-level indention here is only a single space, we should update it to two

ie
```
metadata:
  name: minio-{{.Name}}
  namespace: {{.Namespace}}
...etc...
```